### ISSUE TITLE
gimbal and thrust 'tweakables' available in flight with no connection

### ISSUE THREAD

**Issue by gnivler** [RemoteTechnologiesGroup/RemoteTech#587]
Without any connection, it seems possible to:

adjust the gimbal or thrust sliders
lock and unlock the gimbal

It's not possible to "Shutdown Engine" however so this seems like those actions should be subjected to the same limitations (a bug).

**Comment by gnivler** [comment]
I think this might be contained in RemoteTech.FlightComputer.UIPartActionMenuPatcher

but I have zero idea how to debug the parsing that's going on in there.  Maybe the code predates tweakables?

**Comment by neitsa** [comment]
I took a look at this one and it's way more complicated than I thought at first glance.

Thing is, the "Shutdown Engine" button in the RMB context is a `UIPartActionButton` (`public class UIPartActionButton : UIPartActionEventItem`) which has an event `KSPEvent`. 
On the other hand the "gimbal" button, for example, is a `UIPartActionToggle` (`UIPartActionToggle : UIPartActionFieldItem`) which makes it a `KSPField`.

Simply put, all the machinery in `RemoteTech.FlightComputer.UIPartActionMenuPatcher` is wired for events, not for fields... And we have nothing yet to handle those KSPFields tweakables...

I'll see what I can do to add those tweakables in the delay stack.

**Comment by neitsa** [comment]
Fixed in 59c021f.
